### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
- run rekres
- siderolabs/pkgs to v1.8.0-alpha.0-41-ga97d58f
- golang.org/x/sys to v0.23.0
- linux firmware to 20240709
- google/gvisor to 20240729.0
- cloud-provider-aws to v1.30.3
- containerd-shim-spin to v0.15.1
- kata-containers/kata-containers to 3.7.0
- qemu to 9.0.2
- glib to 2.81.1
- siderolabs/talos-vmtoolsd to v0.6.0
- tailscale/tailscale to 1.70.0

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,7 +64,6 @@ TARGETS += drbd
 TARGETS += ecr-credential-provider
 TARGETS += fuse3
 TARGETS += gasket-driver
-TARGETS += crun
```
Comment (frezbo) [inline Makefile:67]: hmm weird, probably forgot to add in `.kres.yaml`

File: hack/release.toml [hunk 2]
```diff
@@ -40,13 +40,13 @@ ZFS: 2.2.4
...
-qemu-guest-agent: 9.0.1
+ecr-credential-provider: 1.30.3
+qemu-guest-agent: 9.0.2
 mdadm: 4.3
 Intel microcode: 20240531
-Linux firmware: 20240513
+Linux firmware: 20240709
 Spin: 1.5.0
-Gvisor: 20240624.0
+Gvisor: 20240729.0
```
Comment (frezbo) [inline hack/release.toml:49]: other updates missing here?

File: Pkgfile [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+# syntax = ghcr.io/siderolabs/bldr:v0.3.2
 
 format: v1alpha2
 
 vars:
-  LINUX_FIRMWARE_VERSION: "20240513" # update this when updating PKGS_VERSION in Makefile
+  LINUX_FIRMWARE_VERSION: "20240709" # update this when updating PKGS_VERSION in Makefile
   DRBD_DRIVER_VERSION: 9.2.9 # update this when updating PKGS_VERSION in Makefile
   ZFS_DRIVER_VERSION: 2.2.4 # update this when updating PKGS_VERSION in Makefile
   UTIL_LINUX_VERSION: 2.40.1 # update this when updating PKGS_VERSION in Makefile
```
Comment (smira) [inline Pkgfile:9]: I think this got updated in `pkgs` as well?

File: Makefile (hunk 1 above)
Comment (DmitriyMV) [reply Makefile:67]: Added to `.kres.yaml`

File: hack/release.toml (hunk 2 above)
Comment (DmitriyMV) [reply hack/release.toml:49]: Added more.

File: hack/release.toml [hunk 4]
```diff
@@ -39,18 +39,21 @@ Intel Management Engine (IME) modules is now shipped as a Talos System Extension
...
+Linux firmware: 20240709
+Spin: 0.15.1
+Gvisor: 20240729.0
 Wasmedge: v0.4.0
-Kata Containers: 3.6.0
+Kata Containers: 3.3.0
 NVIDIA container toolkit: v1.15.0
 iscsi-tools: v0.1.5
+vmtoolsd: v0.6.0
+glib: 2.81.1
```
Comment (frezbo) [inline hack/release.toml:55]: we can ignore `glib` since it's not shipped directly as an extension, also it;s `util-linux-tools` for the extension name

**Review by frezbo — APPROVED** [review]

File: hack/release.toml (hunk 4 above)
Comment (DmitriyMV) [reply hack/release.toml]: Done!

File: Pkgfile (hunk 3 above)
Comment (DmitriyMV) [reply Pkgfile]: Yep, thanks!

File: container-runtime/gvisor/pkg.yaml [hunk 5]
```diff
@@ -9,8 +9,8 @@ steps:
       # find the go-branch specific merge commit ("Merge release-... (automated)") which has the release-tagged commit as a parent
       - url: https://github.com/google/gvisor/archive/dfeb44ecf5accd9039f5b31a988f6dc39e2bd557.tar.gz
```
[the remark this answers is not in the document]
Comment (DmitriyMV) [reply container-runtime/gvisor/pkg.yaml]: duh! Fixed.

**Pull request merged**